go.mod

```
@@ -70,6 +70,6 @@ require (
 )
 
 replace (
-	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220214200702-86341886e292 // CVE-2021-43565
-	k8s.io/client-go => k8s.io/client-go v0.23.3
+	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
```

I think we should preserve the CVE notation if this replacement is still needed.
```suggestion
	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa  // CVE-2021-43565
```